src/stream/stream_backend.rs

```diff
@@ -136,6 +137,26 @@ fn check_scheme(
                     return Err(SimpleError::new(format!("Endpoint with udp265 scheme only supports H265 encode. Encode: {:?}, Endpoints: {:#?}", encode, endpoints)));
                 }
             }
+            "turn" => {
```

Comment:
@joaoantoniocardoso what was the conclusion of that ?

Reply:
Oh, that the user could use turn or stun, right ?

Reply:
Supposedly yes, but I hadn't implemented that yet. The idea is that the user could pass an arbitrary stun and/or turn servers to be used instead of our local one. Now we have the signalling server too, which, in theory, could also be created by the user if it uses the [`protocol` specified here](https://github.com/centricular/webrtcsink/tree/main/protocol).

To implement this behavior we would need to use our local default unless it's specified by the user, and also identify as a webrtc stream for if a `stun://`, `turn://` or `ws://` is present in the list of endpoints. What do you think?

Reply:
Okay, I did the changes, take a look now - the client can just use `webrtc://` (without host/ip) to use our local servers, or he can specify arbitrary turn, stun, and signalling servers to be used instead.